pkg/remotewrite/config.go

```diff
@@ -201,6 +223,17 @@ func GetConsolidatedConfig(jsonRawConf json.RawMessage, env map[string]string, a
 		return null.NewBool(false, false), nil
 	}
 
+	getEnvMap := func(env map[string]string, prefix string) map[string]string {
+		result := make(map[string]string)
+		for ek, ev := range env {
+			if strings.HasPrefix(ev, prefix) {
+				k := strings.TrimPrefix(ek, prefix)
```

Comment:
```suggestion
		for ek, ev := range env {
			if strings.HasPrefix(ek, prefix) {
				k := strings.TrimPrefix(ek, prefix)
```
I believe this is a typo ^^